app/models/agents/twitter_user_agent.rb

```diff
@@ -95,18 +105,30 @@ def check
       opts = {:count => 200, :include_rts => include_retweets?, :exclude_replies => exclude_replies?, :include_entities => true, :contributor_details => true}
       opts.merge! :since_id => since_id unless since_id.nil?
 
+      if choose_home_time_line?
+
       # http://rdoc.info/gems/twitter/Twitter/REST/Timelines#user_timeline-instance_method
```

Comment:
It looks like this comment is now misplaced, I would be fine with just removing it.